pkg/gardenadm/cmd/token/delete/delete.go

```diff
@@ -49,7 +66,19 @@ gardenadm token delete foo123`,
 	return cmd
 }
 
-func run(_ context.Context, opts *Options) error {
-	opts.Log.Info("Not implemented")
+func run(ctx context.Context, opts *Options) error {
+	clientSet, err := tokenutils.CreateClientSet(ctx, opts.Log)
+	if err != nil {
+		return fmt.Errorf("failed creating client set: %w", err)
+	}
+
+	for _, tokenID := range opts.TokenIDs {
+		if err := clientSet.Client().Delete(ctx, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: bootstraptokenutil.BootstrapTokenSecretName(tokenID), Namespace: metav1.NamespaceSystem}}); client.IgnoreNotFound(err) != nil {
+			return fmt.Errorf("failed deleting bootstrap token secret with ID %q: %w", tokenID, err)
+		}
+
+		fmt.Fprintf(opts.Out, "bootstrap token %q deleted\n", tokenID)
```

Comment:
Should we omit or change this log if the token doesn't exist to prevent confusion?